### DISCUSSION TITLE
New minor release 1.2.0 🥳

### DISCUSSION THREAD

**Discussion by mitrotasios** [tremorlabs/tremor-npm#216] [Announcements]
Hi all, we just released a new minor version that includes multiple improvements to our components which were mostly requested and implemented by the community.

Among other features, it is now possible to select a single Date in the `Datepicker` as well as choosing between a `donut` and `pie` variant for the `DonutChart`.

Thanks everyone for contributing! 🎉

**Comment by Mrahmani71** [comment]
Can you add buttons with type "submit"?

**Reply by mitrotasios** [reply]
Yup, will allow adding intrinsic props such as `className` and `type` to all components soon
